.github/workflows/ci-aqua-security-trivy-tests.yml

```diff
@@ -1,6 +1,26 @@
 name: ci-aqua-security-trivy-tests
 on:
   push:
+    paths-ignore:
+      - '.vscode/CODEOWNERS'
+      - '.vscode/*.json'
+      - 'compose/**'
+      - 'contrib/**'
+      - 'dgraphtest/**'
```

Comment:
this should be removed - if we make a change to our test suite, we want to see if the tests pass or fail